crates/kitsune_p2p/bootstrap/src/num.rs

```diff
@@ -0,0 +1,23 @@
+use crate::store::Store;
+
+use super::*;
+use warp::Filter;
+
+pub(crate) fn num(
+    store: Store,
+) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
```

Comment:
maybe this could be a more generic "stats" api, that returns a struct that for now only has num_peers in it?